tests/submission_test.py

```diff
@@ -327,13 +329,21 @@ def test_related_records(app, admin_idx):
         # First two are valid, and relate to each other
         # 3 has invalid record entry (a string), 4 has invalid data DOI string (doesn't match regex)
         test_data = [
-            {"dir" : "related_submission_1", "related" : 2},
-            {"dir" : "related_submission_2", "related" : 1},
-            {"dir" : "related_submission_3", "related" : None},
-            {"dir": "related_submission_4", "related": None}
+            {"dir": "related_submission_1",
+                "related": 2,
+                "record_title": "Title 1",
+                "expected_title": "Title 2",
+                "expected_version": 1},
+            {"dir": "related_submission_2",
+                "related": 1,
+                "record_title": "Title 2",
+                "expected_title": "Title 1",
+                "expected_version": 1},
+            {"dir": "related_submission_3", "related": None, "record_title": "Title 3"},
+            {"dir": "related_submission_4", "related": None, "record_title": "Title 4"}
         ]
         # Dummy record data
-        record = {'title': 'HEPData Testing',
+        record = {'title': None,
```

Comment:
What was the rationale for replacing the title with `None`?

Reply:
Jordan's comment: "The title will be set later based on `test_data` values."